Data/PackageData.cs

```diff
@@ -125,7 +125,14 @@ public class PackageData
         string RepositoryUrlWithoutGit => RepositoryUrl.EndsWith(".git") ?
             RepositoryUrl.Substring(0, RepositoryUrl.Length - 4) :
             RepositoryUrl;
-        string CombinedRepositoryUrlAndCommit => RepositoryUrlWithoutGit + "/tree/" + RepositoryCommit;
+        string CombinedRepositoryUrlAndCommit {
+            get {
+                if (RepositoryUrl.StartsWith ("https://dev.azure.com/"))
+                    return RepositoryUrlWithoutGit + "?version=GC" + RepositoryCommit;
```

Comment:
Your issue says `version=GBcommit`, but this has `version=GCcommit`; which is correct?